beacon-chain/dberr/BUILD.bazel

```diff
@@ -0,0 +1,9 @@
+load("@prysm//tools/go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["errors.go"],
+    importpath = "github.com/prysmaticlabs/prysm/beacon-chain/dberr",
+    visibility = ["//visibility:public"],
```

Comment:
I think we will get some pushbacks here for visibility to not be public

Reply:
package is gone, resolving this